### ISSUE TITLE
ROS Package name differs from repository name

### ISSUE THREAD

**Issue by flydragon2018** [vincentmllr/deep-robot-grasping#2]
git clone -b noetic-devel https://github.com/vincentmllr/panda_moveit_config.git
Cloning into 'panda_moveit_config'...
fatal: unable to access 'https://github.com/vincentmllr/panda_moveit_config.git/': Failed to connect to github.com port 443: Connection refused

**Comment by vincentmllr** [comment]
I see that the repository is set to `public`, so you should be able to clone. Could it be a local problem with your setup? Was someone else able to clone the repository?

**Comment by flydragon2018** [comment]
thanks.  local connection problem. 

Is " deep-robot-grasping"   equivalent to "panda-deep-grasping " ? 
 
I think the document should be updated .  

I  am still get lost about directory  relationship.

**Comment by vincentmllr** [comment]
Yes, the repository name was `panda-deep-grasping` and is now `deep-robot-grasping`. But the ROS package is still called `panda-deep-grasping`. I am not sure if that also has to be adjusted. Can you check and create a pull request once it is working? It definitly has to be changed in the `README` when cloning the repository.
Maybe its enough if you checkout of the repository and change the local repository folders name to `panda_deep_grasping`.
